backend/app/environment_platform_service_configurations/schema.py

```diff
@@ -0,0 +1,29 @@
+import json
+from typing import Union
+from uuid import UUID
+
+from pydantic import field_validator
+
+from app.environment_platform_service_configurations.glue_schema import AWSGlueConfig
+from app.environment_platform_service_configurations.s3_schema import AWSS3Config
+from app.environments.schema import Environment
+from app.platform_services.schema import PlatformService
+from app.platforms.schema import Platform
+from app.shared.schema import ORMModel
+
+configs = Union[AWSS3Config, AWSGlueConfig]
```

Comment:
```suggestion
ConfigsType = AWSS3Config | AWSGlueConfig
```

This variable should start with an upper-case letter.